bin_center_x: 1.0425
bin_center_y: -0.03
bin_center_z: 0.526

bin_size_x: 0.4
bin_size_y: 0.2
bin_size_z: 0.04

x_rotation: 0.0
y_rotation: 0.0
z_rotation: 0.0

roll_default: 0
pitch_default: 3.14
yaw_default: 0

roll_range: 0.7
pitch_range: 0.7
yaw_range: 6.2

approach_distance: 0.10
deapproach_height: 0.10

step_x: 0.066
step_y: 0.067
step_z: 0.027

step_roll: 0.3
step_pitch: 0.3
step_yaw: 0.6

start_pose: [1.6, 0.04, -1.39, 0.0, -1.46, 0.00]
